WooCommerce/WooCommerceTests/ViewRelated/OrderPaymentDetailsViewModelTests.swift

```diff
@@ -23,9 +25,14 @@ final class OrderPaymentDetailsViewModelTests: XCTestCase {
 
         anotherBrokenOrder = MockOrders().unpaidOrder()
         anotherBrokenOrderViewModel = OrderPaymentDetailsViewModel(order: anotherBrokenOrder)
+
+        orderWithFees = MockOrders().orderWithFees()
+        orderWithFeesViewModel = OrderPaymentDetailsViewModel(order: orderWithFees)
```

Comment:
nit: I think we might want to DI `CurrencySettings` to match the settings used for the fees value? If the previously launched store has different currency settings from `CurrencySettings()` the tests might not pass

```suggestion
        orderWithFeesViewModel = OrderPaymentDetailsViewModel(order: orderWithFees, currencySettings: CurrencySettings())
```

Reply:
Good catch, thanks